scripts/linters/pylint_extensions_test.py

```diff
@@ -2370,6 +2370,877 @@ def test_checks_single_char_and_newline_eof(self):
+                u"""
+                suggestion.change.new_value = (  # pylint: disable=line-too-long type: ignore[attr-defined]
+                    new_content
+                )
+                #@
+                """
+            )
+        node_with_type_ignore.file = filename
+
+        message = testutils.Message(
+            msg_id='mypy-ignore-used',
+            line=2,
+            node=node_with_type_ignore
+        )
+        with self.checker_test_object.assertAddsMessages(message):
+            self.checker_test_object.checker.visit_module(
+                node_with_type_ignore
+            )
+        temp_file.close()
+
+    def test_type_ignores_with_comments_should_not_raises_error(self):
+        node_function = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test'
+        )
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                # Here we use MyPy ignore because attributes on BaseChange
+                # class are defined dynamically.
+                suggestion.change.new_value = (  # type: ignore[attr-defined]
+                    new_content
+                )
+
+                # Here we use MyPy ignore because this function is can only
+                # str values but here we are providing integer which causes
+                # MyPy to throw an error. Thus to avoid the error, we used
+                # ignore here.
+                func_only_accept_str(1234)  # type: ignore[arg-type] #@
+                """
+            )
+        node_function.file = filename
+
+        with self.checker_test_object.assertNoMessages():
+            self.checker_test_object.checker.visit_module(node_function)
+        temp_file.close()
+
+    def test_untyped_call_type_ignores_should_not_raise_error(self):
+        node_function = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test'
+        )
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                # Here we use MyPy ignore because attributes on BaseChange
+                # class are defined dynamically.
+                suggestion.change.new_value = (  # type: ignore[attr-defined]
+                    new_content
+                )
+
+                func_only_accept_str(1234)  # type: ignore[no-untyped-call] #@
+                """
+            )
+        node_function.file = filename
+
+        with self.checker_test_object.assertNoMessages():
+            self.checker_test_object.checker.visit_module(node_function)
+        temp_file.close()
+
+    def test_raises_error_if_gap_in_ignore_and_comment_is_more_than_fifteen(
+        self
+    ):
+        node_with_ignore_and_more_than_ten_gap = astroid.scoped_nodes.Module(
```

Comment:
This is still `more_than_ten` ditto elsewhere

Reply:
Done!